proto/viam/app/cloudslam/v1/cloud_slam.proto

```diff
@@ -0,0 +1,94 @@
+syntax = "proto3";
+
+package viam.app.cloudslam.v1;
+
+import "common/v1/common.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "go.viam.com/api/app/cloudslam/v1";
+
+service CloudSLAMService {
+  rpc StartMappingSession(StartMappingSessionRequest) returns (StartMappingSessionResponse);
+  rpc GetActiveMappingSessionsForRobot(GetActiveMappingSessionsForRobotRequest) returns (GetActiveMappingSessionsForRobotResponse);
+  rpc GetMappingSessionPointCloud(GetMappingSessionPointCloudRequest) returns (GetMappingSessionPointCloudResponse);
+  rpc ListMappingSessions(ListMappingSessionsRequest) returns (ListMappingSessionsResponse);
+  rpc StopMappingSession(StopMappingSessionRequest) returns (StopMappingSessionResponse);
+}
+
+// StartMappingSession
+
+message StartMappingSessionRequest {
+  // The config required to run slam. This includes a replay sensor and map package, and any fields specific to the slam algorithm.
+  google.protobuf.Struct slam_config = 1;
+  // Version to use for slam, defaults stable
+  string slam_version = 2;
+  string map_name = 3;
+}
+
+message StartMappingSessionResponse {
+  string session_id = 1;
+}
+
+// GetActiveMappingSessionsForRobot
+
+message GetActiveMappingSessionsForRobotRequest {
+  // assumes only one active mapping session on a robot
+  string robot_id = 1;
+}
+
+message GetActiveMappingSessionsForRobotResponse {
+  string session_id = 1;
+}
+
+// GetMappingSessionPointCloud
+
+message GetMappingSessionPointCloudRequest {
+  string session_id = 1;
+}
+
+message GetMappingSessionPointCloudResponse {
+  // url to the pointcloud map
+  string url = 1;
```

Comment:
[minor] map_url

Reply:
done!

Reply:
If we deviate from the scope with any names, @nroede could you leave a comment on the scope? (It's ok to deviate if we think of a better name)

Reply:
yes will do now!